src/components/contact-us/contact-us-questions-validation.ts

```diff
@@ -117,15 +121,6 @@ export function validateContactUsQuestionsRequest(): ValidationChainFunc {
           { value, lng: setLanguageToReflectSupportForWelsh(req.i18n.lng) }
         );
       }),
-    body("moreDetailDescription")
```

Comment:
Hi @BeckaL, do we not need this `isLength` max length validation any more? I think `moreDetailDescription` is still present on other forms.

Reply:
An excellent spot! Think this got left in from an earlier reworking that I decided not to do, have added this back in in the relevant commit now so this shouldn't show up in the diff anywhere anymore.